src/AUNewsletterEpic/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { canRender, COMPONENT_NAME } from './canRender';
+import { NewsletterEpic, BrazeMessageProps } from '../NewsletterEpic';
+import { BrazeClickHandler } from '../utils/tracking';
+import { OphanComponentEvent } from '@guardian/types';
+
+const IMAGE_URL =
+    'https://i.guim.co.uk/img/media/9f9f9c06ed5a323b13be816d5c160728c81d1bf9/0_0_784_784/784.png?width=196&s=4c4d5ff2c20821a6e6ff4d25f8ebcc16';
```

Comment:
I think we should be using the `master` crop as the source and then letting the image resizer do its job. (So I'd expect the source URL to have `master` in it somewhere - I think we're using a Grid crop rather than the master crop here).